apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/pathAssertions.kt

```diff
@@ -4,6 +4,7 @@ import ch.tutteli.atrium.creating.Expect
 import ch.tutteli.atrium.domain.builders.ExpectImpl
 import ch.tutteli.atrium.domain.builders.path
 import java.nio.file.Path
+import ch.tutteli.niok.fileNameWithoutExtension
```

Comment:
Add this just for the javadoc, otherwise I'll get a warning but now I get a warning that it's unused import.

Reply:
Seems like a bug of intellij. Gradle does not emit a warning. Hence, keep the import and ignore the warning